scrapinghub/client.py

```diff
@@ -200,7 +201,7 @@ class Project(object):
     """
 
     def __init__(self, client, projectid):
-        self.id = projectid
+        self.key = projectid
```

Comment:
@vshlapakov don't forget to ensure that key is str `self.key = str(projectid)`

and add line with `>>> project.key` output